examples/jose/update.py

```
@@ -0,0 +1,365 @@
+        
+        # Compute document ID based on content
+        self.original_doc_id = compute_mdhash_id(self.original_content, prefix="doc-")
+        
+        # Check if document exists in database
+        doc_exists = await self.rag.full_docs.get_by_id(self.original_doc_id)
+        
+        if doc_exists:
+            logger.info(f"Found original document with ID: {self.original_doc_id}")
+            return self.original_doc_id
+        else:
+            logger.warning(f"Original document not found in database")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)